README.md

```diff
@@ -90,6 +90,20 @@ Please, look at [Examples](EXAMPLES.md) document.
 
 # Asynchronous API
 Each API method has its' async version with *Async* postfix, using *multiprocessing.pool.ThreadPool*.
+Please, look at [Examples](EXAMPLES.md) document
+
+# Aspose Cloud-hosted service VS on-premise deployment (*experimental feature*)
```

Comment:
@s-zubov Please, move this section to correspond with other SDKs e.g. https://github.com/aspose-imaging-cloud/aspose-imaging-cloud-dotnet/tree/development

Reply:
@MaxCrank Thank you, I've moved this section.

Reply:
@s-zubov It still doesn't look like the one from .NET or other SDKs. Please, inspect the .NET SDK readme https://raw.githubusercontent.com/aspose-imaging-cloud/aspose-imaging-cloud-dotnet/development/README.md and take note on headers

> # Examples
> Please, look at [Examples](EXAMPLES.md) document.
> 
> ### Aspose Cloud-hosted service VS on-premise deployment
> Starting from v19.7, you can choose either to use Aspose Cloud-hosted image processing service (the standard way) or the Docker image from Docker Hub deployed on-premise to serve the requests.
> The details about key differences and deployment process is described on the dedicated Docker Hub page.
> 
> To succeed with your on-premise service usage by the SDK, you need to:
> 1. Use the new API class constructor with base URL required parameter, API version and debug mode optional parameters.
> ```csharp
> var imagingApi = new ImagingApi("yourServiceUrl");
> ```
> 2. Set *storage* or *storageName* parameters for each request where they're present (mandatory!).
> 
> # Tests
> Tests are intended for internal usage only.